pkg/querier/http.go

```diff
@@ -11,6 +10,10 @@ import (
 	"strings"
 	"time"
 
+	"github.com/grafana/loki/pkg/loghttp"
+	"github.com/grafana/loki/pkg/loghttp/legacy"
+	v1 "github.com/grafana/loki/pkg/loghttp/v1"
```

Comment:
You could probably import this as the following to improve readability:

```go
import (
  loghttp_legacy "github.com/grafana/loki/pkg/loghttp"
  loghttp_v1 "github.com/grafana/loki/pkg/loghttp/v1"
)
```

Reply:
Good point.  Fixed